components/UnifiedDocFeed/UnifiedDocFeedContainer.tsx

```diff
@@ -23,58 +20,45 @@ import FeedBlurWithButton from "./FeedBlurWithButton";
 import Loader from "../Loader/Loader";
 import Ripples from "react-ripples";
 import UnifiedDocFeedCardPlaceholder from "./UnifiedDocFeedCardPlaceholder";
-import UnifiedDocFeedMenu from "./UnifiedDocFeedMenu";
+import FeedMenu from "./FeedMenu/FeedMenu";
 import fetchUnifiedDocs from "./api/unifiedDocFetch";
-import ExitableBanner from "../Banner/ExitableBanner";
-import DesktopOnly from "../DesktopOnly";
-import { sortOpts } from "./constants/UnifiedDocFilters";
-
+import { getSelectedUrlFilters } from "./utils/getSelectedUrlFilters";
+import ResearchHubBanner from "~/components/ResearchHubBanner";
 const FeedInfoCard = dynamic(() => import("./FeedInfoCard"), {
   ssr: false,
 });
 
 function UnifiedDocFeedContainer({
-  auth, // redux
-  feed,
   home: isHomePage,
   hub, // selected hub
   hubName,
-  hubState, // hub data of current user
   isLoggedIn,
   loggedIn,
   serverLoadedData, // Loaded on the server via getInitialProps on full page load
   subscribeButton,
 }): ReactElement<"div"> {
   const router = useRouter();
   const routerPathName = router.pathname;
-  const [docTypeFilter, setDocTypeFilter] = useState<string>(
-    getFilterFromRouter(router)
-  );
-  const [subFilters, setSubFilters] = useState({
-    filterBy: sortOpts[0],
-    scope: scopeOptions[0],
-    tags: [] as any,
-  });
-
+  const selectedFilters = useMemo(() => {
+    return getSelectedUrlFilters({
+      query: router.query,
+      pathname: router.pathname,
+    });
+  }, [router.pathname, router.query]);
```

Comment:
Pretty sure there's no need to memoize filters here. 
Updating path will trigger everything within feed to rerender because we are changing the fetch to solely depend on the URL now.

Reply:
You know what, you're right. 

I initially had a different implementation and this is an artifact of it